maestro/agent/helm/templates/metrics-proxy.configmap.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-config
+data:
+  nginx.conf: |
+    worker_processes auto;
+    pid /run/nginx.pid;
+
```

Suggestion: send errors to `stderr` to be picked up by the cluster so we have some visibility over any errors. Error log volume should be very light
```suggestion
error_log /dev/stderr;

```